software/mk/Makefile.verilog

```diff
@@ -38,7 +38,7 @@ endif
 ifeq ($(WAVE),0)
   BSG_SIM_EXE = $(BSG_MACHINE_PATH)/simv
 else
-  BSG_SIM_EXE = $(BSG_MACHINE_PATH)/simv-debug
+  BSG_SIM_EXE = $(BSG_MACHINE_PATH)/simv_debug
```

Comment:
@vb000 Do we want it to be named simv_debug or simv-debug?

Reply:
Some other places assume simv-debug. So better not change if there's no issue with simv-debug...

Reply:
I have no problem with simv-debug. I'll fix it

Reply:
I'm fine if you tested it.

Reply:
I did not. I was under the mistaken impression that you were using simv_debug because I misread a variable in #139

Reply:
So I switched it back